src/lib/plugins/entities.js

```diff
@@ -257,13 +270,19 @@ module.exports.entity=function(entity,serv){
   };
 
   entity.collect = (collectEntity) => {
-    if (entity.type != 'player') serv.emit('error', 'Non-player entity (ttype ' + entity.type + ') cannot collect another entity');
-    else {
+    if (entity.type != 'player'){
+      serv.emit('error', 'Non-player entity (ttype ' + entity.type + ') cannot collect another entity')
+    }
```

Comment:
Well the "else" meant it wouldn't run the collect code if it threw an error. So that means you have to add a return inside the "if" if you're not using "else".